.github/workflows/opa.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           aws/enforce_s3_buckets_encryption.rego;aws/enforce_s3_buckets_encryption_test.rego;aws/enforce_s3_buckets_encryption_mock.json
           aws/enforce_kms_key_names.rego;aws/enforce_kms_key_names.test.rego;aws/enforce_kms_key_names.mock.json
           aws/enforce_iam_instance_profiles.rego;aws/enforce_iam_instance_profiles.test.rego;aws/enforce_iam_instance_profiles.mock.json
+          aws/enforce_ebs_del_on_term.rego;aws/enforce_ebs_del_on_term.test.rego;aws/enforce_ebs_del_on_term.mock.json
```

Comment:
@pgale61 JFYI: every new OPA policy should be explicitly included in a test pipeline. Otherwise, we'll fool ourselves with false-positive tests. 

cc: @petroprotsakh @penja it will be very good to fix this so that we can add more policies faster without being worrying they aren't tested.